web/src/components/storage/ProposalVolumes.jsx

```diff
@@ -56,6 +57,33 @@ const sizeText = (size) => {
   return filesize(size, { base: 2 });
 };
 
+/**
+ * Generates an hint describing which attributes affect the auto-calculated limits.
+ * If the limits are not affected then it returns `null`.
+ * @function
+ *
+ * @param {object} volume - storage volume object
+ * @returns {object} component to display (can be `null`)
```

Comment:
can we be more specific with the documented types? `@returns {ReactComponent} or null` ?
same for _volume_

Reply:
And same for the other `@param {object}` added in this PR